server/models/message.py

```diff
@@ -22,12 +22,16 @@ class ChatMessage(Base):
     data = Column(Text, nullable=False)
 
     chat_id = Column(Integer, ForeignKey('chat.id'))
+    chat = relationship('Chat', foreign_keys=[chat_id])
+
     sent_by = Column(Integer, ForeignKey('user.id'))
+    by = relationship('User', foreign_keys=[sent_by, ])
+
 
     def __init__(self, data, chat, sent_by):
         self.data = data
         self.chat = chat
-        self.sent_by = sent_by
+        self.by = sent_by
```

Comment:
Was it renamed by mistake? self.sent_by -> self.by

Reply:
No, I can rename constructor parameter. But original idea was to make ChatMessage class more similar to Message class

Reply:
I was asking that just because __ repr__ was not changed accordingly.